community/kernel/src/main/java/org/neo4j/graphdb/factory/GraphDatabaseSettings.java

```diff
@@ -838,6 +838,13 @@ public String providerName()
     public static final Setting<String> default_advertised_address =
             setting( "dbms.connectors.default_advertised_address", STRING, "localhost" );
 
+    @Description( "Value of the Access-Control-Allow-Origin header sent over any HTTP or HTTPS " +
+            "connector. This defaults to '*', which allows broadest compatibility but is " +
+            "least secure. Note that any URI provided here limits HTTP/HTTPS access to " +
+            "that URI only." )
+    public static final Setting<String> access_control_allow_origin =
+            setting( "dbms.connectors.access_control_allow_origin", STRING, "*" );
+
```

Comment:
This should be in `ServerSettings`, rather than `GraphDatabaseSettings`; the latter is only settings that apply to embedded mode

Reply:
+1

Reply:
I added this immediately below other `dbms.connectors.*` settings. I can see none of those in `ServerSettings`. Could you explain why this one in particular should move?

Reply:
The new setting is only relevant to REST API / HTTP connector. That is why I think it's better to have it together with other REST API settings, like `dbms.rest.transaction.idle_timeout`. Wdyt?